README.md

```diff
@@ -476,7 +476,7 @@ Here are the metrics settings:
 #### Kernel settings
 
 * `settings.kernel.lockdown`: This allows further restrictions on what the Linux kernel will allow, for example preventing the loading of unsigned modules.
-  May be set to "none" (the default), "integrity", or "confidentiality".
+  May be set to "none" (the default in older [variants](variants/)), "integrity" (the default for newer [variants](variants/)), or "confidentiality".
```

Comment:
Should we list the specific variants?  Or perhaps just list the variants which are still defaulting to "none"?

Reply:
```suggestion
  May be set to "none" (the default in older [variants](variants/), up through aws-k8s-1.19), "integrity" (the default for newer variants), or "confidentiality".
```

Reply:
Should we make the same change the SECURITY-GUIDANCE.md doc?

Reply:
Yeah, I'll update the Security Guidance with your suggestion :+1: